### PR TITLE
Patternlba/dp 26357 fix js error

### PR DESCRIPTION
**Pull request by clairesunstudio**
Screen Shot 2022-11-08 at 3 21 00 PM

### CODE REVIEW

File: packages/patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js [hunk 1]
```diff
@@ -32,16 +31,17 @@ const utilNavNarrowCheck = function() {
   return utilNarrowNav ? (utilNarrowNav.offsetWidth > 0 && utilNarrowNav.offsetHeight > 0) : false;
 };
 
+if (hamburgerMenuContainer) {
 // Define all top level clickable elements with current window width.
 let topLevelClickableItems;
```
Comment (tom-fleming) [inline packages/patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js]:
```suggestion
  // Define all top level clickable elements with current window width.
  let topLevelClickableItems;
```

File: packages/patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js [hunk 2]
```diff
@@ -32,16 +31,17 @@ const utilNavNarrowCheck = function() {
...
+    setTimeout(function timeoutFunction() {// This prevents GT elements get null.
+      selectTopClickableItems(width);
+    }, 1000);
+  });
+  window.addEventListener("resize", function (e) {
+    selectTopClickableItems(body.clientWidth);
+  });
 
 // Open and close the menu
 if (menuButton !== null) {
```
Comment (tom-fleming) [inline packages/patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js]:
```suggestion
  // Open and close the menu
  if (menuButton !== null) {
```

File: packages/patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js [hunk 3]
```diff
@@ -830,3 +830,4 @@ setTimeout(function timeoutFunction() {// This prevents GT elements get null.
     });
   }
 }, 1000);// When it's less than 1000, the iframe is null in full screen display.
+}
```
Comment (tom-fleming) [inline packages/patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js]: @clairesunstudio This curly bracket closes this if statement: `if (hamburgerMenuContainer) {`. See [here](https://github.com/massgov/mayflower/pull/1696/files#diff-7b3a2bd65c30faae06061f8aecab1e5e30c7b2a796b16c5e6b2727afbd1f8f78R34). This includes function declarations as well. Is that intentional?

File: packages/patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js (hunk 2 above)
Comment (clairesunstudio) [reply packages/patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js]: used prettify to format

File: packages/patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js (hunk 1 above)
Comment (clairesunstudio) [reply packages/patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js]: used prettify to format

File: packages/patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js (hunk 3 above)
Comment (clairesunstudio) [reply packages/patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js]: yes, that's intentional, actually, I'm going to also include all the declaration queries. 

We need to figure out a better way to scope JS, many JS issues I see are related to it not being scoped properly.

**Review by tom-fleming — APPROVED** [review]
LGTM

**Pull request merged**